x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_issues_step/fix_issues_step.tsx

```diff
@@ -49,10 +51,45 @@ const FixIssuesStep: FunctionComponent<Props> = ({ setIsComplete }) => {
   );
 };
 
+interface CustomProps {
+  navigateToEsDeprecationLogs: () => void;
+}
+
+const AccessDeprecationLogsMessage = ({ navigateToEsDeprecationLogs }: CustomProps) => {
+  return (
+    <WithPrivileges privileges={`index.${DEPRECATION_LOGS_INDEX}`}>
+      {({ hasPrivileges, isLoading }) => {
+        if (isLoading || !hasPrivileges) {
+          // Don't show the message with the link to access deprecation logs
+          // to users who can't access the UI anyways.
+          return null;
+        }
+
+        return (
+          <FormattedMessage
+            id="xpack.upgradeAssistant.overview.fixIssuesStepDescription"
```

Comment:
This id is the same as the one defined in line [104](https://github.com/elastic/kibana/pull/118722/files#diff-b284012f32740c35fef48f98b2f242faf338b667c0f7a46717e74f90a09252f0R104), won't that generate any problems?

Reply:
Great catch, I fixed it!